src/main/java/core/mvc/JsonView.java

```diff
@@ -1,12 +1,53 @@
 package core.mvc;
 
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.http.MediaType;
+
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.io.UnsupportedEncodingException;
 import java.util.Map;
 
 public class JsonView implements View {
+    private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
+    private static final String UTF_8_ENCODING = "UTF-8";
+
     @Override
     public void render(Map<String, ?> model, HttpServletRequest request, HttpServletResponse response) throws Exception {
+        response.setContentType(MediaType.APPLICATION_JSON_VALUE);
+
+        Object bodyValue = getBodyValue(model);
+        response.setContentLength(getContentLength(bodyValue));
+        response.setCharacterEncoding(UTF_8_ENCODING);
+
+        if(!model.isEmpty()){
```

Comment:
공백, 공백라인 컨벤션 등의 부분들은 습관적으로 반영하시는 것이 좋다고 생각해요 😊
```suggestion
        if (!model.isEmpty()) {
```